src/view/constants/api.js

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export const API_PRODUCTION_URL = "https://platform.adobe.io";
+export const API_STAGE_URL = "https://platform-stage.adobe.io";
+export const PLATFORM_PRODUCTION_URL = "https://platform.adobe.com";
+export const PLATFORM_STAGE_URL = "https://platform-stage.adobe.com";
+export const PLATFORM_INTEGRATION_URL = "https://platform-int.adobe.io";
```

Comment:
You don't need these `PLATFORM_` constants. You also don't need `API_STAGE_URL` because we'll be developing the Alloy extension in production. Ricochet was unique.